### ISSUE TITLE
How long to train the CornerNet_Squeeze?

### ISSUE THREAD

**Issue by ybai62868** [princeton-vl/CornerNet-Lite#35]
Hi, author!
I wanna ask some questions about the training speed of your CornerNet_Squeeze with the default hyper-parameters in your configuration.
How long did you take to get the results (on the coco dataset) in the paper compared with yolo_v3?

**Comment by heilaw** [comment]
We trained CornerNet-Squeeze from scratch and the training took around 2 weeks to finish.

**Comment by lililiiiiiiiiii** [comment]
> We trained CornerNet-Squeeze from scratch and the training took around 2 weeks to finish.

hi,How much is suitable for loss?

**Comment by gtwell** [comment]
Why the validation loss is much larger than the training loss?

**Comment by SeeeeShiwei** [comment]
How long to train the CornerNet_Saccade?

**Comment by tc-qaq** [comment]
@heilaw I want to finetune the CornerNet_Squeeze from coco to pascal voc, and i just want train the classifier, which index or name of layers should be train?

**Comment by gtwell** [comment]
> @heilaw I want to finetune the CornerNet_Squeeze from coco to pascal voc, and i just want train the classifier, which index or name of layers should be train?

hg

**Comment by czhi3** [comment]
> We trained CornerNet-Squeeze from scratch and the training took around 2 weeks to finish.

If I didn't wish trained CornerNet-Saccade from scratch, what kind of network model should I choose as the basic model?
